src/structures/resolvedMember.ts

```diff
@@ -54,4 +58,21 @@ export class ResolvedMember {
   get displayName() {
     return this.nick || this.user.username;
   }
+
+  /** The URL to the member's avatar. */
+  get avatarURL() {
+    return this.dynamicAvatarURL();
+  }
+
+  dynamicAvatarURL(format?: ImageFormat, size?: number) {
+    if(!this.avatar) return this.user.dynamicAvatarURL(format, size);
+    if(!format || !ImageFormats.includes(format.toLowerCase())) {
+      format = this.avatar.startsWith('a_') ? 'gif' : this._creator.options.defaultImageFormat;
+    }
+    if(!size || size < ImageSizeBoundaries.MINIMUM || size > ImageSizeBoundaries.MAXIMUM) {
+      size = this._creator.options.defaultImageSize;
+    }
+
+    return `${CDN_URL}${Endpoints.GUILD_MEMBER_AVATAR(null /* guildID - not available */, this.id, this.avatar)}.${format}?size=${size}`;
```

Comment:
I'm pretty sure this is why tests are failing, you assign null to a string parameter.